.github/workflows/preview.yml

```diff
@@ -0,0 +1,22 @@
+name: 'ecma-262'
```

Comment:
this one won't work on ecma402, deploy previews are currently hardcoded to only work on ecma262. you can remove this file for now.

Reply:
yes, I removed this file for now. I want the preview feature but that's work for another PR.